test/data_map_test.exs

```diff
@@ -0,0 +1,162 @@
+defmodule DataMapTest do
+  use PowerAssert
+
+  test "Insert empty row must fail" do
+    c1 = %Masdb.Schema.Column{is_pk: true,  name: "c1", type: :int}
+    c2 = %Masdb.Schema.Column{is_pk: false, name: "c2", type: :int}
+    c3 = %Masdb.Schema.Column{is_pk: false, name: "c3", type: :int}
+
+    timestamp = Masdb.Timestamp.get_timestamp()
+    inserted_value = %{}
+    inserted_schema = %Masdb.Schema{name: "foo", replication_factor: 0, columns: [c1, c2, c3]}
+    inserted_nodeid = "foo@127.0.0.1"
```

Comment:
That's what I have on my iex
![image](https://cloud.githubusercontent.com/assets/3250155/24626576/c73c1cb4-187f-11e7-82c6-2f6fa30558b6.png)
Well, if it's a symbole it could be important since it's not the same type